ignite/pkg/cosmosgen/templates/root/package.json.tpl

```diff
@@ -19,7 +19,7 @@
     "@cosmjs/proto-signing": "0.27.0",
     "@cosmjs/stargate": "0.27.0",
     "@keplr-wallet/types": "^0.11.3", 
-    "axios": "^0.27.2",
```

Comment:
Why this change?

Reply:
there's an unfortunate conflict with the 0.27 version when using swagger-typescript-api to generate Rest clients that leads to type-checking errors (essentially axios changed the type of AxiosHeader)

It works fine in the template (because the template's skipLibCheck is set to true) but causes issues for people explicitly transpiling it

will adjust once a fix is available